articles/client-platforms/angular2/01-login.md

```diff
@@ -77,14 +77,8 @@ and then in your component's template
   <button class="btn btn-primary btn-margin" (click)="auth.login()" *ngIf="!auth.authenticated()">Log In</button>
   <button class="btn btn-primary btn-margin" (click)="auth.logout()" *ngIf="auth.authenticated()">Log Out</button>
 </div>
-
 ```
-The lock widget will pop up showing a Login form, when you click the Login button.
 
+The lock widget will display the Login pop-up form when the **Login** button is clicked.
```

Comment:
Let's reword this to: "The Lock widget will be displayed when the **Login** button is clicked.